### ISSUE TITLE
kpy.plot_cpdb - plot interacting pair rather than genes

### ISSUE THREAD

**Issue by elise-smith** [zktuong/ktplotspy#71]
Thank you for the great package!
For `kpy.plot_cpdb`, I have been specifying genes of interest, but sometimes it would be more useful to plot specific interacting pairs of ligands/receptors.
Could there be an option to plot the `interacting_pair`, rather than just genes?
Many thanks

**Comment by zktuong** [comment]
hi @elise-smith, sure i can make that happen but may take some time.

To clarify, the behaviour you want is to specify e.g. `["CPI-SC0B5CEA47D", "CPI-SC0C8B7BCBB"]` and it will plot it accordingly? do you still want it to display as the gene names in the final output?

**Comment by zktuong** [comment]
OH wait you mean you just want e.g. `TNF-TNFR`?

**Comment by elise-smith** [comment]
Thank you for your quick reply!
Sorry, I just realised that the `genes` option takes ligand receptor pairs, as well as individual genes, which was what I was looking for
Thanks again

**Comment by zktuong** [comment]
well, kinda. the current way it works is it does a regex grab but it's not specific to a specific interacting pair.

I'm going to make a PR soon in the next 5 min and you can try the new option to specify the specify pair's name.

**Comment by zktuong** [comment]
you can try it now with:

```bash
pip install git+https://www.github.com/zktuong/ktplotspy@add-interacting-pairs-option
```

**Comment by elise-smith** [comment]
Just tested it out on a few interacting pairs and it works well! 
Thank you very much

**Comment by zktuong** [comment]
solved with #72
